### DISCUSSION TITLE
指定UP或标题,标签屏蔽动态和视频

### DISCUSSION THREAD

**Discussion by huajijxs** [the1812/Bilibili-Evolved#2298] [功能建议 / Ideas]
希望能增加这种拉黑功能 , 现在营销号和各种带节奏的层出不穷, B站的拉黑功能虽然能拉黑用户, 但是还是能看到他的视频,希望能根据关键词和用户来进行屏蔽,直接眼不见心不烦

**Comment by FoundTheWOUT** [comment]
#1877

**Comment by satarc** [comment]
同样希望加这功能，包括搜索页面也能屏蔽某些up的视频，我搜个tas单纯看看游戏，结果翻了十多页都总有个永恒唠游戏那种说话瞎扯的，看着就烦，搜了半天也没屏蔽功能，唯一搜到可能有用的bilibili清洁者这插件还没了。
